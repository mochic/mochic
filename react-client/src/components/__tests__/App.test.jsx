import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

it('renders deep', () => {
  const tree = renderer.create(<App />);
  expect(tree).toMatchSnapshot();
});
