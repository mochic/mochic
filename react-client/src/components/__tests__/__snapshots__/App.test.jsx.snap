// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders deep 1`] = `
<div>
  <header>
    <ul
      id="headerButtons"
    >
      <li
        className="navButton"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          home
        </a>
      </li>
    </ul>
  </header>
  <div>
    home page
  </div>
  <footer>
    <div>
      footer component
    </div>
    <a
      href="https://github.com/mochic/mochic"
    >
      repo
    </a>
  </footer>
</div>
`;
